eamena/eamena/models/forms.py

```diff
@@ -114,6 +114,45 @@ def load(self, lang):
                     'GENERAL_DESCRIPTION_TYPE.E55' : Concept().get_e55_domain('GENERAL_DESCRIPTION_TYPE.E55')
                 }
             }
+            
+class FeatureSummaryForm(ResourceForm):
+    @staticmethod
+    def get_info():
+        return {
+            'id': 'summary-feature',
+            'icon': 'fa-align-center',
+            'name': _('Resource Summary'),
+            'class': FeatureSummaryForm
+        }
+
+    def update(self, data, files):
+        self.update_nodes('NAME.E41', data)
+        self.update_nodes('HERITAGE_CLASSIFICATION_TYPE.E55', data)
```

Comment:
A node is missing here, `HERITAGE_FEATURE_USE_TYPE.E55`. Could you please add it to this form, @mradamcox ?